Update on-boarding issue template (#4205) (#4968)
DataBiosphere/azul: .github/ISSUE_TEMPLATE/offboarding.md

# Offboarding

> By creating this ticket, you are requesting a employee/contractor to be

Reviewer: ```suggestion
By creating this ticket, you are requesting an employee to be
```

We struck the contractor term from the onboarding template. So we need to do the same here.

- [ ] Set the `Can manage users (admin)` checkbox for the replacement system administrator in the `azul-prod` group in Terra <sub>or the system administrator is not being replaced</sub>
- [ ] Set the `Can manage users (admin)` checkbox for the replacement system administrator in the `azul-anvil-dev` group in Terra <sub>or the system administrator is not being replaced</sub>
- [ ] Set the `Can manage users (admin)` checkbox for the replacement system administrator in the `azul-anvil-prod` group in Terra <sub>or the system administrator is not being replaced</sub>
- [ ] Sent a request email to the organization administrator (cc: hiring supervisor & system administrator) asking them to: 
    - Remove the offboarded employee/contractor's AWS IAM user account from the `platform-{hca,anvil}-{dev,prod}-{viewer,developer,administrator}` groups in the `gi-gateway` AWS account

>     - (If the system administrator is being replaced) add the replacement system administrator's AWS IAM user account to the `platform-hca-{dev,prod,anvildev,anvilprod}-administrator` IAM groups in the `gi-gateway` AWS account

Reviewer: ```suggestion
    - If the system administrator is being replaced, add the replacement system administrator's AWS IAM user account to the `platform-hca-{dev,prod,anvildev,anvilprod}-administrator` IAM groups in the `gi-gateway` AWS account
```
